test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/auth/multiauth/MultiAuthSigningPropertiesTest.java

```diff
@@ -157,10 +157,12 @@ void shouldThrowErrorWhenNoFallback() {
             MultiauthClient client = MultiauthClient.builder()
                                                     .httpClient(mockHttpClient)
                                                     .region(Region.US_WEST_2)
-                                                    .build();
+                                                    .putAuthScheme(authScheme("aws.auth#sigv4a", () -> {
+                                                        throw new RuntimeException("dependency not available");
+                                                    })).build();
 
             assertThatThrownBy(() -> client.multiAuthWithOnlySigv4a(r -> r.stringMember("")))
-                .hasMessageContaining(CRT_DEPENDENCY_ERROR_MESSAGE);
+                .hasMessageContaining("dependency not available");
```

Comment:
CRT specific error is covered in https://github.com/aws/aws-sdk-java-v2/blob/master/test/crt-unavailable-tests/src/test/java/software/amazon/awssdk/auth/aws/AwsV4aAuthSchemeTest.java. Updated this test to verify resolution behavior